mlflow/entities/span.py

```diff
@@ -42,6 +43,31 @@ class SpanType:
     UNKNOWN = "UNKNOWN"
 
 
+def create_mlflow_span(
```

Comment:
note: This change is to suppress the following warning when creating trace under `with mlflow.tracing.disabled():`.
```
Message: "Failed to start span: The `otel_span` argument for the LiveSpan class must be an instance of trace.Span, but got <class 'opentelemetry.trace.span.NonRecordingSpan'>.. "
```

This is because we always tries to create `LiveSpan` instance even if tracing is disabled. Instead we should create `NoOpSpan`.